lib/realtime/tenants.ex

```diff
@@ -231,7 +231,7 @@ defmodule Realtime.Tenants do
     tenant_topic(external_id, sub_topic, public?)
   end
 
-  def tenant_topic(external_id, sub_topic, false), do: "#{external_id}:private:#{sub_topic}"
+  def tenant_topic(external_id, sub_topic, false), do: "#{external_id}:private-#{sub_topic}"
```

Comment:
```suggestion
  def tenant_topic(external_id, sub_topic, false), do: "#{external_id}-private:#{sub_topic}"
```

Otherwise client could subscribe to topic of `private-topic_name` and receive private channel messages.

With `#{external_id}-private:#{sub_topic}` they cannot.

Reply:
Can you add a specific test case for this?

Reply:
my bad 🤦 will change and yes will add one for security purposes